tests/test_FBbasis2D.py

```diff
@@ -17,133 +16,131 @@
 DATA_DIR = os.path.join(os.path.dirname(__file__), "saved_test_data")
 
 
-# NOTE: Default class values (ie. L=8, dtype=np.float32) are listed here and below to
-# to be picked up by TestCase. This means the default values are tested twice by the Mixins.
-@parameterized_class(
-    ("L", "dtype"),
-    [
-        (8, np.float32),
-        (8, np.float64),
-        (16, np.float32),
-        (16, np.float64),
-        (32, np.float32),
-        (32, np.float64),
-    ],
-)
-class FBBasis2DTestCase(TestCase, Steerable2DMixin, UniversalBasisMixin):
-    L = 8
-    dtype = np.float32
-
-    def setUp(self):
-        self.basis = FBBasis2D((self.L, self.L), dtype=self.dtype)
-        self.seed = 9161341
-
-    def tearDown(self):
-        pass
-
-    def _testElement(self, ell, k, sgn):
+params = [
+    (8, np.float32),
+    (8, np.float64),
+    (16, np.float32),
+    (16, np.float64),
+    (32, np.float32),
+    (32, np.float64),
+]
+
+test_bases = [FBBasis2D(L, dtype=dtype) for L, dtype in params]
+
+
+def show_basis_params(basis):
+    # print descriptive test name for parametrized test
+    # run pytest with option -rA to see explicitly
```

Comment:
Since we're now devoting time to these label strings, please add this option to the pytest area of the developer google doc.

Reply:
I see it, thanks.